File: _site/f.php
<html>
<head><title>Sending...</title></head>

<style>

.heading {
	color: #777;
}

.heading a {
	color: #777;
}

</style>

<body>

<?php

require_once('recaptchalib.php');
  $privatekey = "6LdwpckSAAAAAMRk-UxH336Bq06HnPTusZXjzC2M";
  $resp = recaptcha_check_answer ($privatekey,
                                $_SERVER["REMOTE_ADDR"],
                                $_POST["recaptcha_challenge_field"],
                                $_POST["recaptcha_response_field"]);

function sendForm($s, $m) {
		$headers  = 'MIME-Version: 1.0' . "\r\n";
		$headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n";
		mail("ikelewis678@gmail.com", $s, $m, $headers);
	}

function success($t) {
	if ($t == "order") {
		header('Location: http://kctitle.net/success.php?t=order');
		die();
	} elseif ($t == "contact") {
		header('Location: http://kctitle.net/success.php?t=contact');
		die();
	} else {
		echo "Hmmm... seems we're hung up somewhere. It's not your fault... would you mind calling 509-773-5804? We'll get right on it!";
	}
}

if (!$resp->is_valid) {
// What happens when the CAPTCHA was entered incorrectly
die ("The reCAPTCHA wasn't entered correctly. Go back and try it again." .
     "(reCAPTCHA said: " . $resp->error . ")");
} else {

if ($_POST["f"] == 'order') {
		$out = "<html><head><title>Sending...</title></head><style>.heading {color: #777;}.heading a {color: #777}</style><body><span class=\"heading\">Hi there,<br>You just got a title/escrow order from the website. <br>------</span><br><br>";

		extract($_POST, EXTR_SKIP);


		# OWNER section
		$out .= "<b>Owner:</b><br>
					Name: " . $ownerName . "<br>
					Email: " . $ownerEmail . "<br>
					Phone: " . $ownerPhone . "<br>
					Address: " . $ownerAddress . "<br><br>";

		# BUYER section
		$out .= "<b>Buyer:</b><br>
					Name: " . $buyerName . "<br>
					Email: " . $buyerEmail . "<br>
					Phone: " . $buyerPhone . "<br>
					Address: " . $buyerAddress . "<br><br>";

		# PROPERTY section
		$out .= "<b>Property:</b><br>
					Address: " . $propertyAddress . "<br>
					Parcel Number: " . $parcelNumber . "<br><br>";

		# LEGAL DESCRIPTION section
		$out .= "<b>Legal Description:</b><br>
					<p style=\"font-family: monospace;\">
					STD Owner............ " . $stdOwnersVal . " -- Insured by: " . $stdOwnersIns . "<br>
					Purchasers........... " . $purchasersVal . " -- Insured by: " . $purchasersIns . "<br>
					STD Mortgage......... " . $stdMortgageVal . " -- Insured by: " . $stdMortgageIns . "<br>
					EXT Mortgage......... " . $extMortgageVal . " -- Insured by: " . $extMortgageIns . "<br>
					Litigagtion.......... " . $litigationVal . " -- Insured by: " . $litigationIns . "<br>
					Trustee Sale......... " . $trusteeSaleVal . " -- Insured by: " . $trusteeSaleIns . "<br>
					Lot Book Search...... " . $lotBookSearchVal . " -- Insured by: " . $lotBookSearchIns . "<br>
					Limited.............. " . $limitedVal . " -- Insured by: " . $limitedIns . "
					</p><br><br>";

		$out .= "<b>Send Copies To:</b><br>";

		if ($sendToOwner == "on")
			$out .= "Owner";
		if ($sendToSeller == "on")
			$out .= ", Seller";
		if ($sendToListing == "on")
			$out .= ", Listing Agent";
		if ($sendToLender == "on")
			$out .= ", Lender";
		if ($sendToSelling == "on")
			$out .= ", Selling Agent";

		$out .= "</body></html>";

		$subject = "Title/Escrow Order from " . $ownerName . " [KCTitle.net]";

		sendForm($subject, $out);

		success("order");

	} elseif ($_POST["f"] == 'contact') {

		extract($_POST, EXTR_SKIP);

		$out = "<html><head><title>Sending...</title></head><style>.heading {color: #777;}.heading a {color: #777}</style><body><span class=\"heading\">Hi there,<br>You just got an email from " . $name . " (<a href=\"mailto:" . $email . "\">" . $email . "</a>) via the website. <br>------</span><br>";

		$out .= "<p style=\"font-size: 1.2em;\">" . nl2br($message) . "</p>";

		$out .= "</body></html>";

		$subject = "Message from " . $name . " [KCTitle.net]";

		sendForm($subject, $out);

		success("contact");

	} else {
		echo "Something musta gone wrong. We're sorry. Would you mind calling 509-773-5804? We'll get right on it!";
	}

}
?>

</body>

</html>

File: KCTitle/_site/f.rb
#!/usr/bin/env ruby

require 'cgi'
#require 'pony'
cgi = CGI.new
cgi.header("text/html")

h = cgi.params

message = ""

# find out which form is being submitted
if h['f'].to_s() == "order"

	legalDesc = ""

	legalDesc += "STD Owners..........." + h['stdOwnersVal'].to_s() + " -- Insured by: " + h['stdOwnersIns'].to_s() + "<br>"
	
	legalDesc += "Purchasers..........." + h['purchasersVal'].to_s() + " -- Insured by: " + h['purchasersIns'].to_s() + "<br>"
	
	legalDesc += "EXT Mortgage........." + h['extMortgageVal'].to_s() + " -- Insured by: " + h['extMortgageIns'].to_s() + "<br>"

	legalDesc += "Litigation..........." + h['litigationVal'].to_s() + " -- Insured by: " + h['litigationIns'].to_s() + "<br>"

	legalDesc += "Trustee Sale........." + h['trusteeSaleVal'].to_s() + " -- Insured by: " + h['trusteeSaleIns'].to_s() + "<br>"

	legalDesc += "Lot Book Search......" + h['lotBookSearchVal'].to_s() + " -- Insured by: " + h['lotBookSearchIns'].to_s() + "<br>"

	legalDesc += "Limited.............." + h['limitedVal'].to_s() + " -- Insured by: " + h['limitedIns'].to_s() + "<br>"


	message = "Hi there, <br> 
				You just got an email from " + h['ownerName'].to_s() + 
				" about escrow services. Woohoo!<br>" +
				"-----<br><br>" +
				"<b>Owner</b><br>" +
				"Name:    " + h['ownerName'].to_s() + "<br> " +
				"Email:   " + h['ownerEmail'].to_s() + "<br> " +
				"Phone:   " + h['ownerPhone'].to_s() + "<br> " +
				"Address: " + h['ownerAddress'].to_s() + "<br> " +

				"<br><br>" +

				"<b>Buyer</b><br>" +
				"Name:    " + h['buyerName'].to_s() + "<br> " +
				"Email:   " + h['buyerEmail'].to_s() + "<br> " +
				"Phone:   " + h['buyerPhone'].to_s() + "<br> " +
				"Address: " + h['buyerAddress'].to_s() + "<br> " +

				"<br><br>" +

				"<b>Property</b><br>" +
				"Address:       " + h['propertyAddress'].to_s() + "<br> " +
				"Parcel Number: " + h['parcelNumber'].to_s() + "<br> " +

				"<br><br>" +
				
				"<b>Legal Description</b><br><pre>" +

				legalDesc +

				"</pre><br><br>"

else 
	if h['f'].to_s() == "contact"
		message = "<p style=\"color: #444;\">Hey! You just got an email from the website. Here's what " + h['name'].to_s() +
					" said:</p>" +
					"<p style=\"font-size:1.3em\">\"" + h['message'].to_s() + "\"</p>" +
					"<p style=\"color: #444;\">You can get back with " + h['name'].to_s() + " at his email address: " + h['email'].to_s() + "</p>"
	end
end

#Pony.mail(:to => "ikelewis678@gmail.com", :from => "kctitle@kctitle.net", :subject => "Mail from website", :body => message)